dask/dataframe/core.py

```diff
@@ -7970,11 +7985,14 @@ def _reduction_aggregate(x, aca_aggregate=None, **kwargs):
     return aca_aggregate(x, **kwargs)
 
 
-def idxmaxmin_chunk(x, fn=None, skipna=True):
+def idxmaxmin_chunk(x, fn=None, skipna=True, numeric_only=False):
+    numeric_only_kwargs = (
```

Comment:
You're already normalizing these higher in the stack with `check_numeric_only_valid`. Why not just pass-through as `**kwargs`?

Reply:
I don’t like kwargs in the signature, no other reason. Makes things harder to reason about

Reply:
User-facing functions, sure, but this one is supposed to be internal. I suppose it's the matter of taste, still, doesn't feel like enough reason to normalize it twice.

Reply:
Ah I meant this generally, I try to avoid kwargs whenever possible